.github/policies/resourceManagement.yml

```diff
@@ -3240,6 +3246,31 @@ configuration:
       - addLabel:
           label: question
       description: '[arm] auto assign labels and users based on issue description.'
+    - if:
+        - payloadType: Issues
+        - or:
+            - titleContains:
+                pattern: '[Aa]z account(?!(\s+(clear|get-access-token|list|list-locations|set|show))\b)\s*\b'

```

Comment:
What's the purpose of `\s*\b` at the end of regex?

Reply:
The command can contain one or more tailing whitespace(\s*) and  `\b` is a word boundary assertion, it helps to match whole words rather than substrings within words.

e.g.

Matched: `Az account details`
Matched: `Az account`
Not matched: `Az account set`
Not matched: `Az accountlist`
Not matched: `Az accountother`
Matched: `az account other  `

Reply:
I see. Then this regex can be simplified to `[Aa]z account\b(?!\s+(clear|get-access-token|list|list-locations|set|show)\b)`.

Reply:
yeah, it looks good, if `az account **` should be matched, we don't need to worry about tailings words